tools/scenario/tests/test_runtime_province_fill_audit.py

```
@@ -25,14 +25,21 @@ def test_repository_runtime_fill_debt_is_pinned_for_all_scenarios(self):
             for row in audit_repository(root)
         ]
 
+        # 청주 4縣 재판정(data/curated/han/jurisdiction-commandery-adjudications-v1.json: 45107 西平昌·85385 挺·85505 不其·
+        # 85706 安德)으로 정본 소유권이 군국 보유자를 따라 바뀌었지만 런타임 legacy
+        # 경로(infra han.json 도시 → 프로빈스)는 그대로라 채색 부채가 그 4 프로빈스에서만
+        # 움직였다. 1020: 45107 공융→유비 mismatch +1, 85505 missing→85385 missing.
+        # 1021: canonical −2(85505·85706), 45107 extraRuntime +1. 1030/1031: 45107
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)